lib/fastlane/plugin/mango/helper/docker_commander.rb

```diff
@@ -36,6 +39,23 @@ def self.disconnect_network_bridge(container_name: container_name)
       rescue StandardError
         # Do nothing if the network bridge is already gone
       end
+
+      def self.prune
+        Action.sh('docker system prune -f')
+      end
+
+      def self.handle_thin_pool_exception(&block)
+        begin
+          block.call
+        rescue => exception
```

Comment:
I'm not sure which exception class is it.. will need to catch a node in this state to improve this